src/rigid.rs

```diff
@@ -169,6 +169,24 @@ impl<T: Float + ApproxEq<T>, Src, Dst> RigidTransform3D<T, Src, Dst> {
             .to_transform()
             .pre_transform(&self.rotation.to_transform())
     }
+
+    /// Drop the units, preserving only the numeric value.
+    #[inline]
+    pub fn to_untyped(&self) -> RigidTransform3D<T, UnknownUnit, UnknownUnit> {
```

Comment:
would this be equivalent to multiplying by `Scale<f32, T, UnknownUnit>`? If so, there might be a simpler and more universal way to get to/from untyped

Reply:
I'm note sure I follow, rigid transforms don't have a scale. to/from_untyped is mostly there because you have your properly typed code and sometimes you delegate some work to a library that doesn't use units or doesn't use your units, so you cast the unit away for API compatibility.

I wonder if what you mean by scale is some sort of typed identity transform to reify the notion of casting between units in a more explicit way. Maybe it'd make sense in some context? I think that in the case of casting to the unknown unit all we want is to just give up some type info for API compat reason between some independent pieces of code and to/from_untyped are the simplest and most natural ways in that situation.

Reply:
You are correct, rigid transforms don't have a scale, so `Scale` is inapplicable.